scripts/test/hyriseConsole_test.py

```diff
@@ -15,7 +15,8 @@ def initialize():
 		sys.exit(1)
 
 	build_dir = sys.argv[1]
-	console = pexpect.spawn(build_dir + "/hyriseConsole", timeout=60, dimensions=(200, 64))
+        console = pexpect.spawn("catchsegv " + build_dir + "/hyriseConsole", timeout=60, dimensions=(200, 64))
+        console.logfile_read = sys.stdout
```

Comment:
Please make sure to undo this change before continuing, as it prevents the test from failing.